install_config/configuring_authentication.adoc

```diff
@@ -1306,9 +1626,63 @@ described in the OpenID spec. Must use `https`.
 described in the OpenID spec. Must use `https`.
 ====
 
+. Configure which link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[claims] to use: 
++
+----
+claims:
+        id: <4>
+        - custom_id_claim
+        - sub
+        preferredUsername: <5>
+        - preferred_username
+        - email
+        name: <6>
+        - nickname
+        - given_name
+        - name
+        email: <7>
+        - custom_email_claim
+        - email
+----
++
+.. At least one claim to use as the user's identity. The standard identity claim is `sub`.
```

Comment:
sentence fragment, not connected to the line in the example by a footnote reference